test/models/host_test.rb

```diff
@@ -266,6 +266,22 @@ def teardown
       host.expects(:enable_orchestration!).never
       assert host.import_facts(raw['facts'])
     end
+
+    test 'should import facts boot time to reporte data facet' do
+      refute Host.find_by_name('sinn1636.lan')
+      raw = read_json_fixture('facts/facts_with_certname.json')
+      host = Host.import_host(raw['name'], 'puppet')
+      assert host.import_facts(raw['facts'])
+      first_boot_time = host.reported_data.boot_time
+      assert_present host.reported_data.boot_time
+      assert_kind_of DateTime, host.reported_data.boot_time
+
+      sleep 1
```

Comment:
I hope this is was just for debugging. Note that rails allows you to [travel in time](https://api.rubyonrails.org/v5.2.3/classes/ActiveSupport/Testing/TimeHelpers.html#method-i-travel).